api-server/src/common/models/user.js

```diff
@@ -26,7 +26,7 @@ import {
   renderSignInEmail
 } from '../utils';
 
-import { blocklistedUsernames } from '../../server/utils/constants.js';
+import { blocklistedUsernames } from '../../../../config/constants';
```

Comment:
Is this change intentional?

Reply:
Since the client will be using some part of the constants to determine if a user is on a landing page, I thought it would be best to keep the constants in the config.

Reply:
Has the file been moved?

Reply:
yes

Reply:
yes